src/sentry/static/sentry/app/components/eventOrGroupHeader.jsx

```diff
@@ -43,15 +44,22 @@ class EventOrGroupHeader extends React.Component {
     const basePath = `/organizations/${orgId}/issues/`;
 
     if (includeLink) {
+      const locationWithProject = {...browserHistory.getCurrentLocation()};
```

Comment:
We should use `location` from props, since this component gets wrapped with `withRouter`, it should have `location` available. This will make it a bit easier to test: instead of mocking `browserHistory`, we can just pass in `location` in props.

Reply:
Done